.travis.yml

```diff
@@ -0,0 +1,12 @@
+matrix:
+  include:
+    - language: ruby
+      node_js: 
+        - lts
+      install:
+        - gem install bundler
+        - bundler install
+        - npm install --no-optional
```

Comment:
Does `npm ci` work? If we don't do it on the Bamboo build we probably should

Reply:
![image](https://user-images.githubusercontent.com/2552726/50102745-bd046600-021d-11e9-9bab-9d6761930caa.png)